src/wallet-account.ts

```diff
@@ -139,32 +140,53 @@ export class WalletConnection {
         failureUrl?: string
     ) {
         let options: SignInOptions;
-        if (typeof contractIdOrOptions === 'string') {
-            const deprecate = depd('requestSignIn(contractId, title)');
-            deprecate('`title` ignored; use `requestSignIn({ contractId, methodNames, successUrl, failureUrl })` instead');
-            options = { contractId: contractIdOrOptions, successUrl, failureUrl };
+        if (typeof contractIdOrOptions === "string") {
+            const deprecate = depd("requestSignIn(contractId, title)");
+            deprecate(
+                "`title` ignored; use `requestSignIn({ contractId, methodNames, successUrl, failureUrl })` instead"
+            );
+            options = {
+                contractId: contractIdOrOptions,
+                successUrl,
+                failureUrl,
+            };
         } else {
             options = contractIdOrOptions as SignInOptions;
         }
 
         const currentUrl = new URL(window.location.href);
         const newUrl = new URL(this._walletBaseUrl + LOGIN_WALLET_URL_SUFFIX);
-        newUrl.searchParams.set('success_url', options.successUrl || currentUrl.href);
-        newUrl.searchParams.set('failure_url', options.failureUrl || currentUrl.href);
-        if (options.contractId) {            
+        newUrl.searchParams.set(
```

Comment:
This formatting should probably be in a different PR.

Reply:
fixed in the next commit.